File: example/src/main/java/ru/virgil/example/order/BuyingOrderDto.java
package ru.virgil.example.order;

import lombok.Data;
import lombok.EqualsAndHashCode;
import ru.virgil.example.system.IdentifiedDto;

@EqualsAndHashCode(callSuper = true)
@Data
public class BuyingOrderDto extends IdentifiedDto {

    private String description;

}

File: example/src/main/java/ru/virgil/example/order/BuyingOrderDto.kt
package ru.virgil.example.order

import ru.virgil.example.system.dto.IdentifiedDto
import java.time.LocalDateTime
import java.util.*

class BuyingOrderDto(
    override var createdAt: LocalDateTime?,
    override var updatedAt: LocalDateTime?,
    override var uuid: UUID?,
    var description: String?,
) : IdentifiedDto
